Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from twisted.web import http, resource
+
+from Tribler.community.multichain.community import MultiChainCommunity
+
+
+class MultichainEndpoint(resource.Resource):
+    """
+    This endpoint is responsible for handing requests for multichain data.
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+
+        child_handler_dict = {"stats": MultichainStatsEndpoint}
+
+        for path, child_cls in child_handler_dict.iteritems():
+            self.putChild(path, child_cls(session))
+
+
+class MultichainStatsEndpoint(resource.Resource):
+    """
+    This class handles requests regarding the tunnel community debug information.
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+        self.session = session
+
+    def get_multichain_community(self):
+        """
+        Search for the multichain community in the dispersy communities.
+        """
+        for community in self.session.get_dispersy_instance().get_communities():
+            if isinstance(community, MultiChainCommunity):
+                return community
+        return None
+
+    def render_GET(self, request):
+        """
+        .. http:get:: /multichain/stats
```

Comment:
Minor detail: I would spell out `stats` to `statistics`, both in the URL and JSON response.